### PR TITLE
Feature/modal cleanup

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))~~
~~Base axe-core JEST test included for any new LWC (No critical violations)~~
~~CRUD/FLS is enforced in Apex~~
~~Update & Test permission sets to account for new features and CRUD/FLS~~
- [x] All modified classes are unit tested (Apex) at 100% coverage
- [x] UX approval or UX not necessary
~~PR contains draft release notes~~
- [x] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by jjbennett** [comment]
[W-8081262](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008InyoIAC/view)

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -264,6 +264,13 @@
         <shortDescription>No Service Participants Created Warning</shortDescription>
         <value>No Contacts are selected, so no Service Participants will be created.</value>
     </labels>
+    <labels>
+        <fullName>No_Service_Sessions_Created_Warning</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>No Service Sessions Created Warning</shortDescription>
+        <value>No Sessions were added, so no Service Sessions will be created.</value>
```
Comment (jjbennett) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:272]: @lbeller can you please review this new label, similar to the label we already had above on line 265. Thank you!

**Review by bmuniswamy — APPROVED** [review]
Awesome work @jjbennett

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (lbeller) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:272]: Can we just lower case sessions in the first part of the sentence (No sessions were added, so no Service Sessions will be created.)

**Review by lbeller — CHANGES_REQUESTED** [review]
Request minor change to lowercase the first instance of sessions: No sessions were added, so no Service Sessions will be created.

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (jjbennett) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml]: @lbeller - change has been made. Thank you!

**Review by lbeller — APPROVED** [review]

**Pull request merged**